acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/KeyLoadAndSignOpenApiValidationTest.java

```diff
@@ -12,37 +12,35 @@
  */
 package tech.pegasys.eth2signer.tests;
 
+import static io.restassured.RestAssured.given;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Collections.singletonMap;
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.hamcrest.text.IsEqualIgnoringCase.equalToIgnoringCase;
 
 import tech.pegasys.artemis.bls.BLS;
 import tech.pegasys.artemis.bls.BLSKeyPair;
 import tech.pegasys.artemis.bls.BLSPublicKey;
 import tech.pegasys.artemis.bls.BLSSecretKey;
 import tech.pegasys.artemis.bls.BLSSignature;
 import tech.pegasys.eth2signer.dsl.HashicorpSigningParams;
-import tech.pegasys.eth2signer.dsl.HttpResponse;
 import tech.pegasys.eth2signer.dsl.signer.SignerConfigurationBuilder;
 import tech.pegasys.eth2signer.dsl.utils.MetadataFileHelpers;
 import tech.pegasys.signers.bls.keystore.model.KdfFunction;
 import tech.pegasys.signers.hashicorp.dsl.DockerClientFactory;
 import tech.pegasys.signers.hashicorp.dsl.HashicorpNode;
 
 import java.nio.file.Path;
-import java.util.HashMap;
-import java.util.Map;
 import java.util.concurrent.ExecutionException;
 
-import io.netty.handler.codec.http.HttpResponseStatus;
-import io.vertx.core.json.Json;
+import io.restassured.http.ContentType;
+import io.vertx.core.json.JsonObject;
 import org.apache.tuweni.bytes.Bytes;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.EnumSource;
 
-public class KeyLoadAndSignAcceptanceTest extends AcceptanceTestBase {
+public class KeyLoadAndSignOpenApiValidationTest extends AcceptanceTestBase {
```

Comment:
Why rename these tests to include OpenApiValidation? These are testing more than just openAPI validation. Also we should be including AcceptanceTest in the test name to make clear that these are ATs.